test/e2e/ibmcloud_test.go

```diff
@@ -502,6 +502,13 @@ func TestIBMCreateNginxDeployment(t *testing.T) {
 	doTestNginxDeployment(t, assert)
 }
 
+func TestExtraPods(t *testing.T) {
```

Comment:
All the other test cases are have one test content related name, can we change this function name to
`TestServiceAndExtraPod`

Reply:
And this test is good for other providers, so maybe we add this test case in other provider test file too, at last include it in the libvirt_test.go, `TestLibvirtServiceAndExtraPod` ?

Reply:
How about rename to `TestPodToServiceCommunication`?